modules/apps/transfer/keeper/msg_server_test.go

```diff
@@ -106,27 +177,48 @@ func (suite *KeeperTestSuite) TestMsgTransfer() {
 			res, err := suite.chainA.GetSimApp().TransferKeeper.Transfer(ctx, msg)
 
 			// Verify events
-			actualEvents := ctx.EventManager().Events().ToABCIEvents()
-			expectedEvents := sdk.Events{
-				sdk.NewEvent(
-					types.EventTypeTransfer,
-					sdk.NewAttribute(sdk.AttributeKeySender, suite.chainA.SenderAccount.GetAddress().String()),
-					sdk.NewAttribute(types.AttributeKeyReceiver, suite.chainB.SenderAccount.GetAddress().String()),
-					sdk.NewAttribute(types.AttributeKeyAmount, coin.Amount.String()),
-					sdk.NewAttribute(types.AttributeKeyDenom, coin.Denom),
-					sdk.NewAttribute(types.AttributeKeyMemo, "memo"),
-				),
-			}.ToABCIEvents()
+			events := ctx.EventManager().Events().ToABCIEvents()
 
-			if tc.expPass {
+			var expEvents []abci.Event
+			if tc.multiDenom {
```

Comment:
we could probably simplify by just expecting the event attribute to be msg.Tokens